.gitignore

```diff
@@ -1,42 +1,42 @@
-*.dll
-*.exe
-.DS_Store
-example.tf
-terraform.tfplan
-terraform.tfstate
-bin/
-modules-dev/
-/pkg/
-website/.vagrant
-website/.bundle
-website/build
-website/node_modules
-.vagrant/
-*.backup
-./*.tfstate
-.terraform/
-*.log
-*.bak
-*~
-.*.swp
-.idea
-*.iml
-*.test
-*.iml
-.vscode
-
-website/vendor
-
-# Test exclusions
-!command/test-fixtures/**/*.tfstate
-!command/test-fixtures/**/.terraform/
-scripts/testing/auth
-scripts/testing/certs
-
-# build outputs
-results
-dist
-
-# testing
-testing
-testing-mirror/registry.terraform.io/kreuzwerker/docker
+*.dll

```

Comment:
Might only want to change things related to the PR/issue

Reply:
Of course. But I don't see changes in my .gitignore, could be just CRLF issue.